### PR TITLE
add h3

### PR DESCRIPTION
**Pull request by omer826**
first commit to review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,6 +14,8 @@
     <div class="modal">
         <button class="button" onclick="StartGame()">Play</button>
         <h2>Tocuh the number</h2>
+        <h4>week3</h4>
```
Comment (galsinai128) [inline index.html:17]: not good

**Review by galsinai128 — COMMENTED** [review]
check line 17

File: index.html (hunk 1 above)
Comment (omer826) [reply index.html:17]: fixed

**Review by galsinai128 — APPROVED** [review]
done.

**Pull request merged**